### PR TITLE
HEEDLS-517 Fix spacing and lost invalid data in form

### PR DESCRIPTION
**Pull request by AlexJacksonDS**
### JIRA link
https://softwiretech.atlassian.net/browse/HEEDLS-517

### Description
Fixed the spacing between the results count and the cancel link on the Add course to group and Add delegate to group journeys with Javascript off, and fixed data not being preserved with invalid validation for the Complete Within Months

### Screenshots
image
image
image

-----
### Developer checks
(Leave tasks unticked if they haven't been appropriate for your ticket.)

I have:
- [x] Run the formatter and made sure there are no IDE errors.
- [ ] Written tests for the changes (accessibility tests, unit tests for controller, data services, services, view models, etc)
- [x] Manually tested my work with and without JavaScript. Full manual testing guidelines can be found here: https://softwiretech.atlassian.net/wiki/spaces/HEE/pages/6703648740/Testing
- [ ] Updated/added documentation in Swiki and/or Readme. Links (if any) are below:
  - [doc_1_here](link_1_here)
- [x] Updated my Jira ticket with information about other parts of the system that were touched as part of the MR and have to be sanity tested to ensure nothing’s broken.
- [x] Scanned over my own MR to ensure everything is as expected.

### CODE REVIEW

File: DigitalLearningSolutions.Web/ViewModels/TrackingSystem/Delegates/GroupCourses/AddCourseFormData.cs [hunk 1]
```diff
@@ -1,33 +1,34 @@
-﻿namespace DigitalLearningSolutions.Web.ViewModels.TrackingSystem.Delegates.GroupCourses
```
Comment (AlexJacksonDS) [inline DigitalLearningSolutions.Web/ViewModels/TrackingSystem/Delegates/GroupCourses/AddCourseFormData.cs:1]: Fixed line ending in this file

File: DigitalLearningSolutions.Web/ViewModels/TrackingSystem/Delegates/GroupCourses/AddCourseViewModel.cs [hunk 2]
```diff
@@ -1,50 +1,50 @@
-﻿namespace DigitalLearningSolutions.Web.ViewModels.TrackingSystem.Delegates.GroupCourses
```
Comment (AlexJacksonDS) [inline DigitalLearningSolutions.Web/ViewModels/TrackingSystem/Delegates/GroupCourses/AddCourseViewModel.cs:1]: Fixed line endings in this file

**Review by AlexJacksonDS — COMMENTED** [review]
Fixed LF line ending in a couple of files

**Review by DanBloxham-sw — APPROVED** [review]
Looks good to me.

**Review by JBsoftwire — APPROVED** [review]
LGTM

**Pull request merged**